etc/nginx/sites-available/wazo

```diff
@@ -38,5 +38,6 @@ server {
     ssl_certificate /usr/share/xivo-certs/server.crt;
     ssl_certificate_key /usr/share/xivo-certs/server.key;
     ssl_ciphers ALL:!aNULL:!eNULL:!LOW:!EXP:!RC4:!3DES:!SEED:+HIGH:+MEDIUM;
-    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_protocols TLSv1.2 TLSv1.3;
+    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload";
```

Comment:
I don't think we should add `includeSubDomains`, because we can't presume that all subdomains of the Wazo Platform server will be served over HTTPS: they can be hosted on a different machine with totally different constraints.

Also, we should not include `preload` by default, as explained in the [HSTS preload website](https://hstspreload.org/):
> If you maintain a project that provides HTTPS configuration advice or provides an option to enable HSTS, do not include the preload directive by default

Reply:
Thank you for your review @sduthil, modification just done 👍